.gitignore

```diff
@@ -0,0 +1,7 @@
+SAML Bridge.suo
```

Comment:
These first four .suo files are already in the repository. Let's delete these four lines for now, and just add the config-tool paths (including the .suo in that case, since we're not adding it to the repository).